src/SprykerEco/Zed/Econda/EcondaDependencyProvider.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda;
+
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\Econda\Communication\Plugin\CategoryPlugin;
+use SprykerEco\Zed\Econda\Communication\Plugin\ProductsPlugin;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToLocaleFacadeBridge;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToPriceProductFacadeBridge;
+use SprykerEco\Zed\Econda\Dependency\Facade\EcondaToPropelFacadeBridge;
+
+class EcondaDependencyProvider extends AbstractBundleDependencyProvider
+{
+    protected const FACADE_LOCALE = 'FACADE_LOCALE';
+    protected const FACADE_PROPEL = 'FACADE_PROPEL';
+    protected const FACADE_PRICE_PRODUCT = 'FACADE_PRICE_PRODUCT';
+    protected const QUERY_CONTAINER_PRODUCT_IMAGE = 'QUERY_CONTAINER_PRODUCT_IMAGE';
+    protected const QUERY_CONTAINER_PRODUCT_CATEGORY = 'QUERY_CONTAINER_PRODUCT_CATEGORY';
+    protected const FILE_PLUGINS = 'FILE_PLUGINS';
+    protected const PRODUCTS = 'products';
```

Comment:
see suggestion in : https://github.com/spryker-eco/econda/pull/1/files#diff-6370d6728b905d928d1252bab2a6da50R80